symfit/core/minimizers.py

```diff
@@ -316,6 +316,16 @@ def _pack_output(self, ans):
                 fit_results['hessian_inv'] = ans.hess_inv
         return FitResults(**fit_results)
 
+    @classmethod
+    def minimize_method(cls):
```

Comment:
`minimizer_name`? `name`? `algorithm_name`?

Reply:
changed to `method_name`, good point that the original name was not great.